documentation/book/con-kafka-listeners.adoc

```diff
@@ -148,7 +151,89 @@ listeners:
 
 For more information on using node ports to access Kafka, see xref:proc-accessing-kafka-using-nodeports-{context}[].
 
-.Customizing advertised addresses on external listeners
+ifdef::Kubernetes[]
+=== Exposing Kafka using {KubernetesName} `Ingress`
+
+An external listener of type `ingress` exposes Kafka by using {KubernetesName} `Ingress` and the {NginxIngressController}.
+A dedicated `Ingress` resource is created for every Kafka broker pod.
+An additional `Ingress` resource is created to serve as a Kafka bootstrap address.
+Kafka clients can use these `Ingress` resources to connect to Kafka on port 443.
+
+NOTE: External listeners using `Ingress` have been currently tested only with the {NginxIngressController}.
+
+{ProductName} uses the TLS passthrough feature of the {NginxIngressController}.
+Make sure TLS passthrough is enabled in your {NginxIngressController} deployment.
+For more information about enabling TLS passthrough see {NginxIngressControllerTLSPassthrough}.
+Because it is using the TLS passthrough functionality, TLS encryption will be always used when exposing Kafka using `Ingress`.
```

Comment:
just a suggestion ... but maybe it's only me, I would add " ... TLS encryption on the Kafka side ..." in order to be more precise to explain that TLS passthrough means that Nginx will not handle TLS but in any case on Kafka you have to have it enable because the TLS handshake will happen there.

Reply:
Does it really matter who actually handles it? You still have to use the TLS. The main point of this is to make it clear that you cannot disable TLS because of the TLS passthrough. Should it be maybe changed that way? 

_Because it is using the TLS passthrough functionality, TLS encryption cannot be disbled when exposing Kafka using `Ingress`._

Reply:
Sounds better ... even if I like to be explicit with the reader to drive him understanding that anyway the TLS will be handled by Kafka. But if you like this way, no need to change it's really a nit.